src/schemas/CertificateOfCompletionCredential/1-0-0.ts

```diff
@@ -0,0 +1,26 @@
+import { JSONSchema7 } from "json-schema";
+import BaseVerifiableCredential from "../BaseVerifiableCredential/1-0-0";
+
+export default {
+  ...BaseVerifiableCredential,
+
+  $schema: "http://json-schema.org/draft-07/schema#",
+  $id: "https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/CertificateOfCompletionCredential/1-0-0.json",
+
+  title: "Certificate of Completion Credential",
+
```

Comment:
Can you add a description field here and on the star sign cred to describe what the credential is used for. We want to start adding that to all credentials. 
https://github.com/discoxyz/disco-schemas/blob/main/src/schemas/StarSignCredential/1-0-0.ts#L11

Here's an example:
https://github.com/discoxyz/disco-schemas/blob/main/src/schemas/InvestorCredential/1-0-0.ts#L11